source/library/Monadoc/Action/CronEntry/Prune.hs

```diff
@@ -15,10 +15,10 @@ import qualified Witch
 
 run :: App.App ()
 run = do
-  let toKeep = Set.fromList $ Maybe.mapMaybe CronEntry.guid CronEntry.all
-  cronEntries <- App.Sql.query_ "select * from cronEntry where guid is not null"
+  let toKeep = Set.fromList $ Maybe.mapMaybe (.guid) CronEntry.all
+  cronEntries <- App.Sql.query_ @CronEntry.Model "select * from cronEntry where guid is not null"
```

Comment:
This is one of the main downsides. Previously everything was monomorphic, so I the types were inferred. But with the overloaded dots, everything is polymorphic. That means I have to specify which type I want returned from these queries. Arguably that's a good thing, but it does make things a bit noisier (especially with joins).